packages/nodes-base/nodes/Hubspot/CompanyDescription.ts

```diff
@@ -896,15 +896,14 @@ export const companyFields = [
 					loadOptionsMethod: 'getCompanyProperties',
 				},
 				default: '',
-				description: `Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your companys. Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by ,.`,
+				description: `<p>Used to include specific company properties in the results. By default, the results will only include company ID and will not include the values for any properties for your companies.</p><p>Including this parameter will include the data for the specified property in the results. You can include this parameter multiple times to request multiple properties separated by a comma: <code>,</code>.</p>`,
 			},
 			{
 				displayName: 'Properties With History',
 				name: 'propertiesWithHistory',
 				type: 'string',
 				default: '',
-				description: `Works similarly to properties=, but this parameter will include the history for the specified property,
-				instead of just including the current value. Use this parameter when you need the full history of changes to a property's value.`,
+				description: `Works similarly to properties=, but this parameter will include the history for the specified property, instead of just including the current value. Use this parameter when you need the full history of changes to a property's value.`,
```

Comment:
`properties=` is this a typo or does it mean something? not familiar enough with hubspot to know

Reply:
<img width="587" alt="image" src="https://user-images.githubusercontent.com/44588767/143258612-0bfa3b4f-7ebe-4669-995a-b7f6f7a79572.png">
https://legacydocs.hubspot.com/docs/methods/deals/get-associated-deals